app/src/main/res/layout/activity_child_registration.xml

```diff
@@ -4,141 +4,244 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:orientation="vertical" android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context="amazingme.activities.app.ChildRegistrationActivity">
+    tools:context="amazingme.activities.app.ChildRegistrationActivity"
+    android:background="@drawable/upperbackgroundlb">
 
-    <TextView
-        android:id="@+id/child_registration_date_of_birth_text_view"
-        android:layout_width="145dp"
-        android:layout_height="41dp"
-        android:text="Date of Birth"
-        android:textAlignment="center"
-        android:textColor="@color/amazing_me_blue"
-        android:textSize="24sp"
-        android:layout_marginTop="17dp"
-        app:layout_constraintRight_toRightOf="@+id/child_registration_last_name"
-        app:layout_constraintTop_toBottomOf="@+id/child_registration_last_name"
-        tools:layout_editor_absoluteX="35dp" />
+    <EditText
+        android:id="@+id/editText2"
```

Comment:
can't have these kind of names laying around... it needs to be more specific (like it was). 

i think all that should have changed for this, really, were the constraints. what other changes did you make?